pkg/io/io.go

```diff
@@ -21,3 +22,14 @@ var GetFile = func(host, src, target string) ([]byte, error) {
 	}
 	return f, nil
 }
+
+// FetchEnv Fetch env vars from the OCP3 cluster
```

Comment:
I think it made sense in Config but I'm sure there's a reason for this move.

Reply:
`FetchEnv ` is here because it's not directly related to application config.